package/boot/uboot-sunxi/patches/100-sun6i-alternate-on-UART2.patch

```diff
@@ -6,11 +6,11 @@ Subject: sun6i: define alternate-function for UART2 on GPG
 
 --- a/arch/arm/include/asm/arch-sunxi/gpio.h
 +++ b/arch/arm/include/asm/arch-sunxi/gpio.h
-@@ -190,6 +190,7 @@ enum sunxi_gpio_number {
+@@ -171,6 +171,7 @@ enum sunxi_gpio_number {
+ #define SUN5I_GPG_SDC1		2
  #define SUN6I_GPG_SDC1		2
  #define SUN8I_GPG_SDC1		2
- #define SUN6I_GPG_TWI3		2
-+#define SUN6I_GPG_UART2         2
++#define SUN6I_GPG_UART2	2
+ #define SUN8I_GPG_UART1		2
```

Comment:
This patch is only used by the next one `package/boot/uboot-sunxi/patches/101-sun6i-support-console-on-UART2.patch` you can remove this too if the remove `package/boot/uboot-sunxi/patches/101-sun6i-support-console-on-UART2.patch`